app/src/main/java/mozilla/lockbox/store/AutoLockStore.kt

```diff
@@ -80,6 +80,12 @@ class AutoLockStore(
             // push lockrequired if required
             .map { lockCurrentlyRequired() }
             .subscribe(autoLockActivated as Subject)
+
+        autoLockActivated
+            .filter { it }
+            .map { DataStoreAction.Lock }
+            .subscribe(dispatcher::dispatch)
```

Comment:
similarly, in iOS we ended up having the datastore read the appropriate lock state and take action accordingly, rather than dispatch from a store.

Reply:
are we going to be able to remove the `AccountStore` -> `DataStoreAction` part of the `RootPresenter` because of this?

Reply:
...ah I see you did that